actors/builtin/miner/policy.go

```diff
@@ -310,37 +294,8 @@ var RewardVestingSpec = VestSpec{ // PARAM_SPEC
 }
 
 // When an actor reports a consensus fault, they earn a share of the penalty paid by the miner.
-// This amount is:  Min(initialShare * growthRate^elapsed, maxReporterShare) * collateral
-// The reward grows over time until a maximum, forming an auction for the report.
-func RewardForConsensusSlashReport(elapsedEpoch abi.ChainEpoch, collateral abi.TokenAmount) abi.TokenAmount {
-	// High level description
-	// var growthRate = SLASHER_SHARE_GROWTH_RATE_NUM / SLASHER_SHARE_GROWTH_RATE_DENOM
-	// var multiplier = growthRate^elapsedEpoch
-	// var slasherProportion = min(INITIAL_SLASHER_SHARE * multiplier, 0.05)
-	// return collateral * slasherProportion
-
-	// BigInt Operation
-	// NUM = SLASHER_SHARE_GROWTH_RATE_NUM^elapsedEpoch * INITIAL_SLASHER_SHARE_NUM * collateral
-	// DENOM = SLASHER_SHARE_GROWTH_RATE_DENOM^elapsedEpoch * INITIAL_SLASHER_SHARE_DENOM
-	// slasher_amount = min(NUM/DENOM, collateral)
-	elapsed := big.NewInt(int64(elapsedEpoch))
-
-	// The following is equivalent to: slasherShare = growthRate^elapsed
-	// slasherShareNumerator = growthRateNumerator^elapsed
-	slasherShareNumerator := big.Exp(consensusFaultReporterShareGrowthRate.Numerator, elapsed)
-	// slasherShareDenominator = growthRateDenominator^elapsed
-	slasherShareDenominator := big.Exp(consensusFaultReporterShareGrowthRate.Denominator, elapsed)
-
-	// The following is equivalent to: reward = slasherShare * initialShare * collateral
-	// num = slasherShareNumerator * initialShareNumerator * collateral
-	num := big.Mul(big.Mul(slasherShareNumerator, consensusFaultReporterInitialShare.Numerator), collateral)
-	// denom = slasherShareDenominator * initialShareDenominator
-	denom := big.Mul(slasherShareDenominator, consensusFaultReporterInitialShare.Denominator)
-
-	// The following is equivalent to: Min(reward, collateral * maxReporterShare)
-	// Min(rewardNum/rewardDenom, maxReporterShareNum/maxReporterShareDen*collateral)
-	return big.Min(big.Div(num, denom), big.Div(big.Mul(collateral, consensusFaultMaxReporterShare.Numerator),
-		consensusFaultMaxReporterShare.Denominator))
+func RewardForConsensusSlashReport(epochReward abi.TokenAmount) abi.TokenAmount {
+	return big.Div(epochReward, big.NewInt(consensusFaultReporterDefaultShare))
```

Comment:
Although I found out about this after my initial stab at this, I did notice that the math here is different from [there](https://github.com/filecoin-project/specs-actors/pull/1354/files#diff-25d448ee2846a665efe3de55eab372d83c4a4ef4b27108a0250b03439c6b48f0R289-R292). Unsure if I'm misinterpreting the spec or maybe it changed since 1354 was opened? Let me know if I'm misunderstanding something and I'll adjust.

Reply:
BTW, kuba, I didn't realize you had a PR until I produced this one. Apologies. 🙇

Reply:
The consensus fault report reward is defined as `BR/4`.
`epochReward = E*BR` with `E=5`.